times_reader/transforms.py

```diff
@@ -2174,6 +2173,7 @@ def convert_aliases(input: Dict[str, DataFrame]) -> Dict[str, DataFrame]:
     return output
 
 
+# TODO: A function like this is also inluded in main.py. Should we delete one?
```

Comment:
@siddharth-krishna @samwebster should we delete the function from here or from main?

Reply:
But for this one, perhaps keep it in main and remove from transforms? This is less a transformation and more of an IO action.
Sam, correct me if I'm missing something.